contracts/protocol/libraries/logic/LiquidationLogic.sol

```diff
@@ -134,6 +121,21 @@ library LiquidationLogic {
       })
     );
 
+    (
+      vars.collateralAToken,
```

Comment:
Neither the `userCollateralBalance` nor any of the return values from `_getConfigurationData()` is used in the `validataLiquidationCall()`. Could be moved down below the validity check to reduce gas when validation reverts.

Reply:
good point